src/screens/LazyLoader/page.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
 import BackgroundLayout from 'components/BackgroundLayout';
-import Loader from 'containers/LazyLoader/loader';
+import Loader from 'screens/LazyLoader/loader';
 
 const Page = () => {
   return (
-    <BackgroundLayout>
+    <BackgroundLayout dynamicImage={ null }>
```

Comment:
@rafayet-monon Can we `dynamicImage` as an optional variable with default value?
I think about that if we don't pass a parameter like this `dynamicImage={ null }` 🤔

Reply:
@Lahphim  `dynamicImage` is optional. We are using it if something is provided in the props. And passing the `null` value to get the black background with no image. We are using a [default image](https://github.com/rafayet-monon/surveyor/blob/chore/task-10-update-code-conventions/src/components/PageBackground/index.js#L6) and overriding it if `dynamicImage` is present. Please share your ideas to restructure it. I couldn't think of a better way. 😞

Reply:
@Lahphim let me know what you think.

Reply:
@Lahphim changed as you suggested.